src/raftstore/store/peer.rs

```diff
@@ -378,8 +378,8 @@ impl Peer {
             proposals: Default::default(),
             apply_proposals: vec![],
             pending_reads: Default::default(),
-            peer_cache: RefCell::new(FlatMap::default()),
-            peer_heartbeats: FlatMap::default(),
+            peer_cache: RefCell::new(HashMap::default()),
```

Comment:
So here we are supposed to be using `fxhash`? `::default()` is enough? `::with_hasher()` is not needed for this?

Reply:
`with_hasher` is not necessary. `HashMap<K, S>` is an alias of `FxHashMap<K, S>`, which is an alias of `std::collections::HashMap<K, V, FxBuildHasher>`.

Reply:
Ah, right, I see the alias now: https://github.com/pingcap/tikv/pull/3137/files#diff-91c2e21eb3bb3e6831f5ec0cc61e41eaL15